scanner.cs

```diff
@@ -3,66 +3,176 @@
 using System.Collections.Generic;
 
 public class Scanner {
-	private int curByte = 0;
-	private byte[] __bytes;
-	private List<Tuple<string, TOKENS>> __tokens;
+    private int curByte = 0;
+    private int column = 1;
+    private int line = 1;
```

Comment:
For consistency, change all private variables to begin with two underscores; Apply this to these three variables above.